roles/gitea/defaults/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+gitea_user: gitea
+gitea_group: gitea
+gitea_home: "/var/lib/gitea"
+gitea_shell: "/bin/false"
+gitea_version: "1.20.3"
+gitea_version_check: true
+gitea_config_dir: "/etc/gitea"
+
+gitea_dl_url: "https://github.com/go-gitea/gitea/releases/download/v{{ gitea_version }}/gitea-{{ gitea_version }}-linux-{{ gitea_arch }}"
+gitea_repository_root: "{{ gitea_home }}"
+gitea_user_repo_limit: -1
+
+gitea_app_name: "Metal Gitea"
+gitea_http_domain: localhost
+gitea_root_url: http://localhost:3000
+gitea_protocol: http
+gitea_http_listen: 0.0.0.0
+gitea_http_port: 3000
+gitea_disable_http_git: false
+gitea_offline_mode: true
+
+gitea_db_type: sqlite3
+gitea_db_path: "{{ gitea_home }}/data/gitea.db"
+
+gitea_ssh_listen: 0.0.0.0
+gitea_ssh_domain: gitea
+gitea_start_ssh: true
+gitea_ssh_port: 2222
+
+gitea_secret_key: T0pS3cr31
+gitea_internal_token: SomethingVeryLong
```

Comment:
Do we need to variablize these?

In the future, should these values be ansible-vault secrets or something?